crates/sui-source-validation/src/tests.rs

```diff
@@ -422,25 +427,15 @@ async fn module_bytecode_mismatch() -> anyhow::Result<()> {
         panic!("Expected verification to fail");
     };
 
-    let SourceVerificationError::ModuleBytecodeMismatch { address, package, module } = err else {
-        panic!("Expected ModuleBytecodeMismatch, got: {:?}", err);
-    };
-
-    assert_eq!(address, b_ref.0.into());
-    assert_eq!(package, "b".into());
-    assert_eq!(module, "c".into());
+    let expected = expect!["Local dependency did not match its on-chain version at <id>::b::c"];
```

Comment:
We are losing some fidelity in these tests, because of how addresses are being sanitized -- suggest either having `sanitize_id` accept a map from addresses to some stable representation, or adding a separate assertion for the address.